repos/fdbt-site/src/pages/api/csvUpload.ts

```diff
@@ -342,7 +344,9 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
                 // put the now updated matching json into s3
```

Comment:
I think this comment might need amending as its not updating S3, its amending the database products table.